dataland-frontend/tests/e2e/fixtures/p2p/P2pPreparedFixtures.ts

```diff
@@ -0,0 +1,86 @@
+import { FixtureData } from "@sharedUtils/Fixtures";
+import { PathwaysToParisData } from "@clients/backend";
+import { generateP2pFixtures } from "./P2pDataFixtures";
+
+/**
+ * Generates Pathway To Paris prepared fixtures by generating random Pathway To Paris datasets and afterwards manipulating some fields
+ * via manipulator-functions to set specific values for those fields.
+ * @param undef_probability probability for a field to be undefined
+ * @param toggleRandomSectors determines if the sector list should include all possible sectors or a randomized selection
+ * @returns the prepared fixtures
+ */
+export function generateP2pPreparedFixtures(
+  undef_probability: number,
+  toggleRandomSectors = true
+): Array<FixtureData<PathwaysToParisData>> {
+  const preparedFixtures = [];
+  preparedFixtures.push(
+    manipulateFixtureForSixP2pDataSetsInDifferentYears(
+      generateP2pFixtures(1, undef_probability, toggleRandomSectors)[0]
+    )
+  );
+  preparedFixtures.push(manipulateFixtureForDate(generateP2pFixtures(1, undef_probability)[0], "2023-04-18"));
```

Comment:
From these three date changed fixtures only "P2p-date-2023-04-18" appears to be used. Can the rest be removed?

Reply:
Checking it

Reply:
Removed two unnecessary ones